profiles/use.desc: add global man flag

epsilon-0 opened the pull request:
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

juippis reviewed (approved):
@gentoo/qa check please.

Dessa commented:
sys-auth/sssd is missing from this list.
i changed manpages to man with my pr but the oldest stable still uses manpages useflag

epsilon-0 commented:
> sys-auth/sssd is missing from this list.
> i changed manpages to man with my pr but the oldest stable still uses manpages useflag

added :D 
keeping the manpages flag until the package is removed.
